rpcd/playbooks/ansible.cfg

```diff
@@ -1,5 +1,8 @@
 [defaults]
 gathering = smart
+fact_caching = jsonfile
```

Comment:
Why do these fact caching lines need adding? What is making use of them in this pull request?